### ISSUE TITLE
training explain in detail info

### ISSUE THREAD

**Issue by john09282922** [naseemap47/YOLO-NAS#13]
Thanks for nice works.
I did well for training. But I'd like to change some hyper parameter, starting lr and weight decay, optimizer, etc.
Could you send me some info to modify them?

thanks,
john

**Comment by naseemap47** [comment]
Hi @john09282922 ,
Thank you for using my repo. I will update these variables to training arguments ASAP.
Thank you for your feedback.
If you any other suggestion, please let me know.

**Comment by naseemap47** [comment]
Hi @john09282922 
Added lr and weight decay, optimizer, etc. to training arguments.
I think this will help.
Let me know your feedback.
Thank you.
